bob/test_main.py

```diff
@@ -23,10 +24,13 @@ def test_echo(self):
     def test_help_command(self):
         main.help_command(update=MockUpdate, context=None)
         self.assertTrue(True)
-    
+
     def test_space_command(self):
-        main.space_command(update=MockUpdate, context=None)
-        self.assertTrue(True)
+        update = MockUpdate
+        main.space_command(update, None)
+        self.assertNotEqual(None, update.message.reply_message_text)
+        self.assertRegex(update.message.reply_message_text,
+                         r"Seuraava.*\n.*Helsinki.*\n.*T-:")
```

Comment:
Maybe could also add test that message includes digits (some days/hours/seconds).